src/wolfgame.py

```diff
@@ -3046,7 +3046,13 @@ def myrole(var, wrapper, message):
         return
     role = evt.data["role"]
 
-    wrapper.pm(messages["show_role"].format(role))
+    badguys = get_all_players(Wolfchat)
```

Comment:
This is not accurate if `var.RESTRICT_WOLFCHAT` is configured, see `src/roles/helper/wolves.py` lines 354-359. Note also that warlock always sees cursed even if they aren't in wolfchat, so should be explicitly listed as well.

If you're sticking with getting all players with those roles instead of just keeping it as a set of roles, add appropriate checks to ensure the role exists (see below comment). get_all_players with a role that doesn't exist in the bot will error. It is always safe to call it with role categories; the issues only occur if you try to hardcode certain roles in there (such as warlock).